barretenberg/cpp/src/barretenberg/translator_vm/goblin_translator_flavor.hpp

```diff
@@ -879,6 +879,13 @@ class GoblinTranslatorFlavor {
      * @brief A container for univariates used during sumcheck.
      */
     template <size_t LENGTH> using ProverUnivariates = AllEntities<bb::Univariate<FF, LENGTH>>;
+    /**
+     * @brief A container for univariates used during Protogalaxy folding and sumcheck with some of the computation
```

Comment:
translator doesn't have anything to do with Protogalaxy (same for eccvm and avm) so I wonder whether this should be here

Reply:
Ok, removed